libs/types/apollo.config.js

```diff
@@ -11,6 +11,12 @@ module.exports = {
       '../../apps/token/client.graphql',
       '../../apps/trading/client.graphql',
     ],
-    excludes: ['**/generic-data-provider.ts', '**/__generated___/*'],
+    excludes: [
+      '**/generic-data-provider.ts',
+      '**/__generated___/*',
+      '../../libs/accounts/**',
```

Comment:
question (non blocking): Why ignore these libs?

Reply:
because they have been migrated, and this is the configuration for the old types gen. we want to stop generating the oldschool types for these. this config file will be removed when all libs and apps have been migrated.